helpers/helpers.v2.1.d/utils

```diff
@@ -228,9 +228,14 @@ _ynh_apply_default_permissions() {
     # App files can have files of their own
     if ynh_system_user_exists --username="$app"; then
         # If this is a file in $install_dir or $data_dir : it should be owned and read+writable by $app only
-        if [ -f "$target" ] && (is_in_dir "$target" "${install_dir:-}" || is_in_dir "$target" "${data_dir:-}" || is_in_dir "$target" "/etc/$app"); then
-            chmod 600 "$target"
-            chown "$app:$app" "$target"
+        if (is_in_dir "$target" "${install_dir:-}" || is_in_dir "$target" "${data_dir:-}" || is_in_dir "$target" "/etc/$app"); then
+            if [ -d "$target" ]; then
+              chmod -R u+rwX "$target"
+              chown -R "$app:$app" "$target"
```

Comment:
It should be $app:$group.

I suggest to adapt `if [ "$target" == "${install_dir:-}" ]; then` below by replacing it with: `if is_in_dir "$target" "${install_dir:-}" ; then`

Same for data_dir (and maybe /etc/$app)